### ISSUE TITLE
Error in all charts after upgrading Angular, Progres ...

### ISSUE THREAD

**Issue by MaklaCof** [telerik/kendo-angular#1633]
## I'm submitting a...
* Regression report
* Bug report

## Current behavior
I update `@angular` and `@progress` npm packages.  Now, I get error if I use `kendo-chart` in my project.

> TypeError: Cannot read property 'runOutsideAngular' of undefined
>     at ThemeService.next (configuration.service.js:50)
>     at ThemeService.push (configuration.service.js:33)
>     at ThemeService.loadTheme (theme.service.js:61)
>     at new ChartComponent (chart.component.js:115)
>     at createClass (core.js:11909)
>     at createDirectiveInstance (core.js:11744)
>     at createViewNodes (core.js:13206)
>     at createEmbeddedView (core.js:13083)
>     at callWithDebugContext (core.js:14491)
>     at Object.debugCreateEmbeddedView [as createEmbeddedView] (core.js:13826)

## Expected behavior
No error, and graphs are rendered.

## Minimal reproduction of the problem with instructions
This [demo code](https://www.telerik.com/kendo-angular-ui-develop/components/charts/chart/#toc-basic-usage) is not working with my packages.

## Environment

Package versions:
```
E:\MyProject.Web>npm ls --depth 0
myproject.web@1.0.0 E:\MyProject.Web
+-- @angular-devkit/build-angular@0.6.8
+-- @angular/animations@6.0.6
+-- @angular/cli@6.0.8
+-- @angular/common@6.0.6
+-- @angular/compiler@6.0.6
+-- @angular/compiler-cli@6.0.6
+-- @angular/core@6.0.6
+-- @angular/forms@6.0.6
+-- @angular/http@6.0.6
+-- @angular/platform-browser@6.0.6
+-- @angular/platform-browser-dynamic@6.0.6
+-- @angular/platform-server@6.0.6
+-- @angular/router@6.0.6
+-- @aspnet/signalr@1.0.0
+-- @fortawesome/angular-fontawesome@0.1.0-10
+-- @fortawesome/fontawesome-svg-core@1.2.0-14
+-- @fortawesome/pro-light-svg-icons@5.1.0-11
+-- @fortawesome/pro-regular-svg-icons@5.1.0-11
+-- @fortawesome/pro-solid-svg-icons@5.1.0-11
+-- @ngtools/webpack@6.0.8
+-- @ngx-translate/core@10.0.2
+-- @ngx-translate/http-loader@3.0.1
+-- @progress/kendo-angular-buttons@4.1.1
+-- @progress/kendo-angular-charts@3.2.1
+-- @progress/kendo-angular-dateinputs@3.4.2
+-- @progress/kendo-angular-dialog@3.6.2
+-- @progress/kendo-angular-dropdowns@3.0.2
+-- @progress/kendo-angular-excel-export@2.1.0
+-- @progress/kendo-angular-grid@3.5.0
+-- @progress/kendo-angular-inputs@3.1.3
+-- @progress/kendo-angular-intl@1.4.1
+-- @progress/kendo-angular-l10n@1.2.0
+-- @progress/kendo-angular-layout@3.1.1
+-- @progress/kendo-angular-menu@0.4.0
+-- @progress/kendo-angular-popup@2.4.1
+-- @progress/kendo-angular-scrollview@2.0.1
+-- @progress/kendo-angular-sortable@1.1.0
+-- @progress/kendo-angular-treeview@2.2.0
+-- @progress/kendo-angular-upload@4.1.2
+-- @progress/kendo-data-query@1.3.1
+-- @progress/kendo-date-math@1.2.0
+-- @progress/kendo-drawing@1.5.6
+-- @progress/kendo-theme-default@2.54.0
+-- @telerik/kendo-intl@1.4.1
+-- @types/chai@4.1.4
+-- @types/gapi@0.0.35
+-- @types/gapi.auth2@0.0.47
+-- @types/jasmine@2.8.8
+-- @types/jest@23.0.0
+-- @types/node@10.3.6
+-- angular2-router-loader@0.3.5
+-- angular2-template-loader@0.6.2
+-- angulartics2@6.1.1
+-- aspnet-prerendering@3.0.1
+-- aspnet-webpack@3.0.0
+-- awesome-typescript-loader@5.0.0
+-- chai@4.1.2
+-- codelyzer@4.3.0
+-- core-js@2.5.7
+-- css@2.2.3
+-- css-loader@0.28.11
+-- event-source-polyfill@0.0.12
+-- expose-loader@0.7.5
+-- file-loader@1.1.11
+-- hammerjs@2.0.8
+-- html-loader@0.5.5
+-- isomorphic-fetch@2.2.1
+-- jasmine-core@3.1.0
+-- jest@22.0.0
+-- jest-preset-angular@5.2.3
+-- json-loader@0.5.7
+-- jwt-decode@2.2.0
+-- node-sass@4.9.0
+-- preboot@5.1.7
+-- raw-loader@0.5.1
+-- reflect-metadata@0.1.12
+-- rimraf@2.6.2
+-- rxjs@6.2.1
+-- rxjs-compat@6.2.1
+-- sass-loader@7.0.3
+-- style-loader@0.21.0
+-- to-string-loader@1.1.5
+-- tslint@5.10.0
+-- typescript@2.7.2
+-- url-loader@1.0.1
+-- webpack-bundle-analyzer@2.13.1
+-- webpack-hot-middleware@2.22.2
+-- webpack-merge@4.1.3
-- zone.js@0.8.26
```

Browser:
- Chrome (desktop) version `Version 67.0.3396.87 (Official Build) (64-bit)`

System:
- TypeScript version: 2.7.2
- Node version: v9.4.0
- Platform:  Windows

**Comment by danielkaradachki** [comment]
I can't replicate the problem in a new project using the same package versions. Please provide a runnable sample that demonstrates the error.

**Comment by MaklaCof** [comment]
Hi, I tracked down error to `tsconfig` `target` setting. If set to `es6` it is not working.
Demo is in zip attachment.

It is basic ASP.NET Core Angular application with modified `tsconfig.json` and `packages.json`. Just open and run. When page stops loading  and nothing is displayed open console.

[WebApplication10.zip](https://github.com/telerik/kendo-angular/files/2137706/WebApplication10.zip)

**Comment by danielkaradachki** [comment]
Thanks, I was able to replicate the error. We will look into it.

**Comment by danielkaradachki** [comment]
Should be fixed in the latest dev version.

**Comment by tsvetomir** [comment]
Shipped in v3.3.0 of the `@progress/kendo-angular-charts` package.
